contrib/windows/julia-manifest.xml

```diff
@@ -7,6 +7,11 @@
       </requestedPrivileges>
     </security>
   </trustInfo>
+  <dependency>
+    <dependentAssembly>
+      <assemblyIdentity type='win32'  name='Microsoft.Windows.Common-Controls' version='6.0.0.0'     processorArchitecture='*' publicKeyToken='6595b64144ccf1df' language='*' />
```

Comment:
merge "... from unknown repository": Github seems to think you've deleted your fork of Julia?

I suspect you'll need to delete this PR and create a new one. I'm not sure that particular action is recoverable.

(impressively, Github's "Merging via command line" have text specific to dealing with this case, since without a branch it can't be cherry-picked or merged. Instead you need `curl -L https://github.com/JuliaLang/julia/pull/22632.patch | git am -3`)